src/lib/loaders/loaders_with_authentication/gravity.js

```diff
@@ -11,6 +11,7 @@ export default (accessToken, userID, opts) => {
   )
 
   return {
+    artworkLoader: gravityLoader(id => `artwork/${id}`),
```

Comment:
this could have unexpected consequences, I'd strongly recommend re-naming it to `authenticatedArtworkLoader`

Reply:
Which has more consequences, in that we would lose caching for artworks if more things moved to use this too - it might be worth pairing with me on this alas :P

Reply:
haha I just changed it to `artworkLoader`. Will change it back. Curious what kind of consequences could happen? cc: @starsirius @mzikherman

Reply:
Looks like it was discussed above, https://github.com/artsy/metaphysics/pull/1149#discussion_r199689878

IMO I think we can add a boolean to the resolver for requesting the private data loader instead, and that could work it out

Reply:
Hah, I just went and said that we needed the same behavior as the first version of this PR.
Using `artworkLoader` for both auth'd an un-auth'd would remove caching on an artwork for all logged in users.

This is fine for the `saleArtworks` because it's pretty likely that the data would differ per user, but an artwork is nearly always the exact same for every user. We shouldn't drop caching for that.

Reply:
I might miss something here, but I'm under the impression that authenticated data loaders [don't  cache beyond the lifecycle of a request](https://github.com/artsy/metaphysics/blob/642ad437388725f5b798de7d952f2971bbf2a776/src/lib/loaders/api/loader_with_authentication_factory.js#L17-L18). Would that still invalidate caches for logged-in users?